server/src/main/codegen/includes/parserImpls.ftl

```diff
@@ -247,14 +247,73 @@ SqlCreate SqlCreateTable(Span s, boolean replace) :
     final SqlIdentifier id;
     SqlNodeList tableElementList = null;
     SqlNode query = null;
+
+    SqlCreate createTableLike = null;
 }
 {
     <TABLE> ifNotExists = IfNotExistsOpt() id = CompoundIdentifier()
-    [ tableElementList = TableElementList() ]
-    [ <AS> query = OrderedQueryOrExpr(ExprContext.ACCEPT_QUERY) ]
+    (
+        <LIKE> createTableLike = SqlCreateTableLike(s, replace, ifNotExists, id) {
+            return createTableLike;
+        }
+    |
+        [ tableElementList = TableElementList() ]
+        [ <AS> query = OrderedQueryOrExpr(ExprContext.ACCEPT_QUERY) ]
+        {
+            return SqlDdlNodes.createTable(s.end(this), replace, ifNotExists, id, tableElementList, query);
+        }
+    )
+}
+
+SqlCreate SqlCreateTableLike(Span s, boolean replace, boolean ifNotExists, SqlIdentifier id) :
```

Comment:
You calling `SqlCreateTableLike` by `SqlCreateTable `.
> `SqlCreateTable` is added to `createStatementParserMethods`

Can we add `SqlCreateTableLike` to `createStatementParserMethods`?

Reply:
Fix this. Could you please review this PR again?